### PR TITLE
pkg/u8g2: update version

### PR DESCRIPTION
**Pull request by PeterKietzmann**
This updates the u8g2 lib package. Second patch needed adaption. I felt so free to merge previous 0002 and 0003 patches. Tested with a PCD8544 and MAX7219

### CODE REVIEW

**Comment by smlng** [comment]
compile tests look good. Btw. there are lots of enhancements since you updated the version, your patches also work with the latest commit (`9833db01f3e8b865af94e40237fa8c937b732159`) as of 22.09.2017. 

So maybe bump version even a bit further? In the mean while I try to find a display and make some more/real tests 😉

File: pkg/u8g2/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PKG_NAME=u8g2
 PKG_URL=https://github.com/olikraus/u8g2
-PKG_VERSION=7346987f4e2eadc2e0791b77a1c1c8a27bd72a93
+PKG_VERSION=e8764381a751c1743c01e9f9ea24fb812ab754d5
```
Comment (smlng) [inline pkg/u8g2/Makefile]: also works for newer version, please pump to `9833db01f3e8b865af94e40237fa8c937b732159`

File: pkg/u8g2/patches/0001-u8g2-add-RIOT-makefiles.patch [hunk 2]
```diff
@@ -72,5 +72,5 @@ index 0000000..32e7913
 +
 +include $(RIOTBASE)/Makefile.base
 -- 
-1.9.1
+2.7.4
```
Comment (smlng) [inline pkg/u8g2/patches/0001-u8g2-add-RIOT-makefiles.patch]: which then would be `2.18.2`

**Comment by PeterKietzmann** [comment]
@smlng thanks! I've updated the version and adapted the version number immediately. Tests with PCD8544 and MAX7219 still succeed

**Comment by smlng** [comment]
can confirm that, too!

**Comment by PeterKietzmann** [comment]
Willing to merge?

**Comment by smlng** [comment]
ACK & GO

**Pull request merged**